tools/load_balancing_tool/README

```diff
@@ -9,352 +9,6 @@
-                        for i=0..#data points (original + extrapolated,
-		            c in [ice,lnd,ocn,atm]
-            all T vars >=0
-	    all N,NB vars integer
-
-This MILP is solved using the PuLP python interface to the COIN-CBC solver
-https://pythonhosted.org/PuLP/
-https://www.coin-or.org/Cbc/
-
-
-************************************
-4. Extending the Load Balancing Tool
-************************************
-The file $CIME_DIR/tools/load_balancing_tool/optimize_model.py
-contains a base class OptimizeModel as well as an implementation class
-IceLndAtmOcn. Any layout solver will look similar to IceLndAtmOcn
-except for the components involved and the layout-specific constraints.
-
-Example class and inherited methods that should be overridden:
-
-file my_new_layout.py:
-#*************************************************
-import optimize_model
-
-class MyNewLayout(optimize_model.OptimizeModel)
-   def get_required_components(self):
-       """
-       Should be overridden by derived class. Return a list of required
-       components (capitalized) used in the layout.
-       Example: return ['ATM', 'LND', 'ICE']
-       """
-
-
-   def optimize(self):
-        """
-        Run the optimization.
-        Must set self.state using LpStatus object:
-              LpStatusOptimal    -> STATE_SOLVED_OK
-              LpStatusNotSolved  -> STATE_UNSOLVED
-              LpStatusInfeasible -> STATE_SOLVED_BAD
-              LpStatusUnbounded  -> STATE_SOLVED_BAD
-              LpStatusUndefined  -> STATE_UNDEFINED
-              -- use self.set_state(lpstatus) --
-        Returns state
-
-        If solved, then solution will be stored in self.X dictionary, indexed
-        by variable name. Suggested convention:
-        'Tice', 'Tlnd', ... for cost per component
-        'Nice', 'Nlnd', ... for ntasks per component
-        'NBice', 'NBlnd', ... for number of blocks per component
-
-        The default implementation of get_solution() returns a dictionary
-        of these variable keys and their values.
-        """
-
-   def get_solution(self):
-       """
-       Return a dictionary of the solution variables, can be overridden.
-       Default implementation returns values in self.X
-       """
-#*************************************************
-
-To use this new layout:
-   1. save the class MyNewLayout in file my_new_layout.py
-   2. make sure that my_new_layout.py is in PYTHONPATH
-   3. Use those names in your execution command line argument to --layout
-      $ ./load_balancing_solve.py ... --layout my_new_layout.MyNewLayout
-
-   -- to permanently add to CIME --
-
-   1. add MyNewLayout class to layouts.py
-   2. run using '--layout MyNewLayout'
-   3. add test in tests/load_balance_test.py that uses that name in command
-      line argument (see test for atm_lnd)
-   4. make pull request
+Complete documentation of this tool is available in the CIME Miscellaneous Tools section.
```

Comment:
Is there a --help message?   Also I think we should indicate the 2 modes in the README.  Something like this:
"This Load Balancing tool finds reasonable PE layouts for CIME-driven models.  It will find these from timing files you provide or from runs done by the tool.

For more information, see the help message or complete documentation in the Miscellaneous Tools section of the CIME user's guide."

Reply:
yes --help works